src/generators/endpoint.ts

```diff
@@ -0,0 +1,181 @@
+import uuidv4 from 'uuid/v4';
+import {names} from './names';
+import {loadDoc} from '../dumper';
+import ora from 'ora';
+
+const command = 'endpoint [options]';
+
+const describe = 'generate endpoint data';
+
+const builder = {
+    endpoint_count: {
+        alias: 'n',
+        describe: 'number of endpoint to populate',
+        default: 100,
+        type: 'number',
+    },
+    doc_count: {
+        alias: 'd',
+        describe: 'number of document per endpoint',
+        default: 50,
+        type: 'number'
+    },
+};
+
+function handler(argv: any): void {
+    const spinner = ora({
+        text: 'Generating endpoint data',
+    });
+    spinner.start();
+    generateDocs(argv);
```

Comment:
should `handler` be an `async` function and this call here `await` for it?

`generatDocs()` returns a Promise, thus if the goal is to stop the spinner after generating the docs, then you will need to `await` here or: 

```javascript
generateDocs(argv).finally(() => spinner.succeed())
```

Or - handle failures as well:

```javascript
generateDocs(argv)
    .then(() => spinner.succeed())
    .then(err => spinner.fail(err.message))
```